### PR TITLE
Vertex AI Experiments - Compare Models - Notebook review

### PR DESCRIPTION
**Pull request by inardini**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/experiments/comparing_local_trained_models.ipynb [hunk 1]
```diff
@@ -398,9 +398,14 @@
         "3. In the **Service account name** field, enter a name, and\n",
```
Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>missing H1 title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>Split into 4 cells</p><p>Move Dataset after Objective</p><p>Use template for Objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>Combine these 2 cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>put install cells into single cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>combine 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>use latest template</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>add some explanation of the code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>Add some explanation of the code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>combine cleanup into single cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Comment (andrewferlitsch) [inline notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>set delete_bucket to False</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>done</p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:398]: <p>done </p>

Comment (inardini) [reply notebooks/official/experiments/comparing_local_trained_models.ipynb:412]: <p>done</p>

**Comment by inardini** [comment]
@andrewferlitsch ready to be merged

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**